src/lib/services/transfer/__tests__/endpoint-manager/__snapshots__/task.spec.ts.snap

```diff
@@ -0,0 +1,170 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`endpointManager.task cancel 1`] = `
+{
+  "headers": {
+    "accept": "*/*",
+    "accept-encoding": "gzip,deflate",
+    "connection": "close",
+    "content-length": "63",
+    "content-type": "application/json",
+    "host": "transfer.api.globusonline.org",
+    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)",
+  },
+  "json": {
+    "message": "a test message",
+    "task_id_list": [
+      "example-task-id",
+    ],
+  },
+  "method": "POST",
+  "url": "https://transfer.api.globusonline.org/v0.10/endpoint_manager/admin_cancel",
+}
+`;
+
+exports[`endpointManager.task get 1`] = `
+{
+  "headers": {
+    "accept": "*/*",
+    "accept-encoding": "gzip,deflate",
+    "connection": "close",
+    "host": "transfer.api.globusonline.org",
+    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)",
+  },
+  "method": "GET",
+  "url": "https://transfer.api.globusonline.org/v0.10/endpoint_manager/task/example-task-id",
+}
+`;
+
+exports[`endpointManager.task getAdminCancel 1`] = `
+{
+  "headers": {
+    "accept": "*/*",
+    "accept-encoding": "gzip,deflate",
+    "connection": "close",
+    "content-length": "0",
+    "host": "transfer.api.globusonline.org",
+    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)",
+  },
+  "json": undefined,
+  "method": "POST",
+  "url": "https://transfer.api.globusonline.org/v0.10/endpoint_manager/admin_cancel/example-admin-cancel-id",
+}
+`;
+
+exports[`endpointManager.task getAll 1`] = `
+{
+  "headers": {
+    "accept": "*/*",
+    "accept-encoding": "gzip,deflate",
+    "connection": "close",
+    "host": "transfer.api.globusonline.org",
+    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)",
+  },
+  "method": "GET",
+  "url": "https://transfer.api.globusonline.org/v0.10/task_list",
+}
+`;
+
+exports[`endpointManager.task getEventList 1`] = `
+{
+  "headers": {
+    "accept": "*/*",
+    "accept-encoding": "gzip,deflate",
+    "connection": "close",
+    "host": "transfer.api.globusonline.org",
+    "user-agent": "node-fetch/1.0 (+https://github.com/bitinn/node-fetch)",
+  },
+  "json": undefined,
+  "method": "GET",
+  "url": "https://transfer.api.globusonline.org/v0.10/endpoint_manager/task/example-task-id/event_list?filters=iserror%3A1",
```

Comment:
`?filters=iserror%3A1"` – this looks incorrect.

<img width="793" alt="Screenshot 2024-01-22 at 11 07 59 AM" src="https://github.com/globus/globus-sdk-javascript/assets/694253/c8351c76-2217-443a-b6b0-c9afb2bf56ce">



https://docs.globus.org/api/transfer/advanced_endpoint_management/#get_task_events

Reply:
Would you expect to pass that as a number or string?

Reply:
A `number` seems ok for now; It's kinda hard to say without an implementation. I do keep on waffling a bit because "Filter Type: `flag`" kinda sounds more like a string than a number... but we can always swap it if need be.

Reply:
Swapped this to a number and updated the call site in the test.